Mathlib/Algebra/GroupPower/CovariantClass.lean

```diff
@@ -357,3 +356,36 @@ theorem one_le_zpow {x : G} (H : 1 ≤ x) {n : ℤ} (hn : 0 ≤ n) : 1 ≤ x ^ n
 #align zsmul_nonneg zsmul_nonneg
 
 end DivInvMonoid
+
+/-!
+### Deprecated lemmas
+
+Those lemmas have been deprecated on the 2023/12/23.
+-/
+
+@[deprecated] alias pow_le_pow_of_le_left' := pow_le_pow_left'
```

Comment:
I think you might actually need
```suggestion
@[deprecated pow_le_pow_left'] alias pow_le_pow_of_le_left' := pow_le_pow_left'
```

Reply:
Gladly not:
* https://github.com/leanprover-community/mathlib4/actions/runs/7307366536/job/19913007492
* https://github.com/leanprover-community/mathlib4/pull/9179/files#diff-f0fe1b51418558c50bf0a4987ea95933f803b45120d299f6566377f3b78baf6cR982